### ISSUE TITLE
empty publication slot

### ISSUE THREAD

**Issue by gglusman** [NCATSTranslator/Feedback#363]
https://ui.test.transltr.io/results?l=Familial%20Pityriasis%20Rubra%20Pilaris&t=0&q=c545963d-9921-4ae5-8b0e-a12a24b3a43f

(-)-epigallocatechin gallate
2 Paths that may treat Familial Pityriasis Rubra Pilaris

The first path is a direct 'treats' link with 6 papers 'supporting'. The first three are about the disease and the CARD14 gene. The next two are about the compound. The last one is empty:
![Screenshot 2023-06-17 at 17 39 55](https://github.com/NCATSTranslator/Feedback/assets/3977332/960b90f2-bf7d-4592-9534-fec6d9bb7b21)
The linkout from the empty publication is to https://ui.test.transltr.io/Unknown and leads to:
Unexpected Application Error!
404 Not Found

The empty publication seems to come from unsecret agent, which also shows an empty publication:
![Screenshot 2023-06-17 at 17 43 03](https://github.com/NCATSTranslator/Feedback/assets/3977332/d6679825-f639-45a6-b49f-56016b13a521)

(The second path states that the compound 'affects' the CARD14 gene, which 'contributes to' the disease. The 'contributes to' part is correct. The 'affects' part appears to have some valid publication support.)

**Comment by Genomewide** [comment]
@gprice1129 I think this is an unsecret issue. Should we assign back to them? I am adding @webyrd to this.

**Comment by dnsmith124** [comment]
You're correct @Genomewide, the issue is coming from the empty PMID attached to that edge so we should reassign to unsecret.

**Comment by dnsmith124** [comment]
In the meantime Gus has implemented a solution to the issue on our end, so the problem shouldn't reoccur for us (once that has been deployed)

**Comment by sierra-moxon** [comment]
retested on CI today: https://ui.ci.transltr.io/results?l=Familial%20Pityriasis%20Rubra%20Pilaris&i=MONDO:0008251&t=0&q=137ebbb1-fd80-45a4-8382-edb5ccd70117

epigallocatechin gallate no longer shows up in the results. 

close?

**Comment by gglusman** [comment]
Might as well since with epigallocatechin gallate no longer showing up, it's hard to evaluate whether the empty publication would have been there if the result were returned. ;)

**Comment by gglusman** [comment]
Actually I'm seeing it there on ci, with a score of 0 and no publications supporting at all:
https://ui.ci.transltr.io/results?l=Familial%20Pityriasis%20Rubra%20Pilaris&i=MONDO:0008251&t=0&q=d4025549-f5bf-470b-8c44-19be43866441

**Comment by sierra-moxon** [comment]
thanks @gglusman - I can't believe I was bitten again by the "finished searching but not finished results" bug.

**Comment by Genomewide** [comment]
The UI shows it has graphs from Aragorn and BTE.  No evidence
![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/93866af9-0f5e-450e-99e1-718753e5636c)

Arax has odd results for the Aragorn and BTE?

BTE
![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/7abcb883-6774-401c-b362-119b141cce1c)

Aragorn
![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/b045b2a4-1554-439d-b30c-15e2048eaab0)

I thought we could click on the support graphs or view them some how?

**Comment by Genomewide** [comment]
Aragorn's in the CI ARAX (forgot to that above)

![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/a420918e-2d04-4dbf-a201-3443a0fb7a30)


And here showing the support graph (I believe they are flexing a bit LOL )
![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/0e31251c-66eb-4a5f-9200-80503e496c6f)

But I don't know why there are only 8 paths in the UI? Which may be a separate but related issue?

![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/14fefb9e-0aa2-4f91-bf3d-ac19659afaa8)
